config/config.go

```diff
@@ -7,17 +7,17 @@ import (
 	"strings"
 	"sync"
 
-	yaml "gopkg.in/yaml.v2"
```

Comment:
Last PR you did, Brian asked to leave that unchanged.

Reply:
yeah nice catch too